### PR TITLE
Contractor Uplink tgui-next overhaul

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Redoes the contractor uplink in full tgui-next with a couple of fun bonus features.
image
(nulls are just because it was on an empty server)
image

Also new feature where every contract has a randomly generated wanted description:
image

Works on the format prefix + verb + noun + location.
Give me suggestions for more nouns, verbs, and locations.

## Changelog
:cl:
add: Reworked the Contractor Uplink for tgui-next
/:cl:

### CODE REVIEW

File: tgui-next/packages/tgui/interfaces/SyndContractor.js [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    "Awaiting response...",
+    "Awaiting response...",
+    "Response received, ack 4851234...",
+    "CONFIRM ACC " + (Math.round(Math.random() * 20000)),
+    "Setting up private accounts...",
+    "CONTRACTOR ACCOUNT CREATED",
+    "Searching for availible contracts...",
+    "Searching for availible contracts...",
+    "Searching for availible contracts...",
+    "Searching for availible contracts...",
```
Comment (Angustmeta) [inline tgui-next/packages/tgui/interfaces/SyndContractor.js]: Typos: "availible"

**Comment by HomocidlePancakes** [comment]
Noun: Catgirl(s)

File: tgui-next/packages/tgui/interfaces/SyndContractor.js (hunk 1 above)
Comment (actioninja) [reply tgui-next/packages/tgui/interfaces/SyndContractor.js]: I can't spell

**Comment by Akrilla** [comment]
big approve very nice

**Comment by GravitronBlue** [comment]
New polish looks nice, don't see the point of the new wanted descriptions as they are now though.
Might be interesting if admins could also create custom reasons, perhaps possible already with varedit.

**Comment by actioninja** [comment]
It's just pointless fluff.  Admins could edit it if they wanted to but that's not really why I did it.

**Review by kriskog — APPROVED** [review]

**Pull request merged**